src/SbManager/Scripts/manager/homeController.js

```diff
@@ -20,7 +20,15 @@
         if (!window.confirm("You sure? This can't be undone and your world might explode.")) return;
         $scope.model = null;
         $.post(window.applicationBasePath + "/api/v1/busmanager/deleteall", function (d) {
-            window.location = "#/";
+            $scope.refresh();
+        });
+    };
+
+    $scope.deleteAllDeadLetters = function () {
+        if (!window.confirm("You sure? This can't be undone and your world might explode.")) return;
```

Comment:
Would be nice to have a different message to avoid confusion between the delete buttons- they have very different consequences